stories/ui/Button.js

```diff
@@ -1,185 +1,82 @@
-/* @flow */
```

Comment:
Can you put this back? I don't see any errors. If you run into an error, you can use [`$FLOW_FIXME` comments](https://github.com/mozilla/addons-frontend#flow).

Reply:
I know that `Button` doesn't have Flow coverage today but hopefully it will soon!

Reply:
> Can you put this back? 

oops! yes, that was inadvertently removed..will this add back.

Reply:
maybe I should set up an issue for Button to get flow coverage?

Reply:
> maybe I should set up an issue for Button to get flow coverage?

Sure!